allennlp/data/tokenizers/sentence_splitter.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+from overrides import overrides
+from allennlp.common import Registrable
+from allennlp.common.util import get_spacy_model
+from allennlp.data.tokenizers.token import Token
+
+
+class SentenceSplitter(Registrable):
+    """
+    A ``SentenceSplitter`` splits strings into sentences. There are two variants of sentence splitting,
```

Comment:
I wouldn't hard-code that there are two variants, because that will eventually become stale and we'll forget to update this.